src/Internal/Payment/PaymentContext.php

```diff
@@ -7,8 +7,12 @@
 
 namespace WCPay\Internal\Payment;
 
+use PHP_CodeSniffer\Tokenizers\PHP;
 use WC_Payments_API_Abstract_Intention;
 use WCPay\Internal\Payment\PaymentMethod\PaymentMethodInterface;
+use WCPay\Internal\Payment\Change;
+
+use function PHPUnit\Framework\isEmpty;
```

Comment:
Not sure how this one and `PHP_CodeSniffer\Tokenizers\PHP` got here, but it seems like a left-over.

Reply:
Copy paste error, removed.